docs/tutorials/data_manipulation/pandasdataframes.md.template

```diff
@@ -123,7 +123,7 @@ from gluonts.dataset.pandas import PandasDataset
 
 max_end = max(df.groupby("item_id").apply(lambda _df: _df.index[-1]))
 dfs_dict = {}
-for item_id, gdf in df.groupby("item_id"):
+for item_id, gdf in df_missing_val.groupby("item_id"):
     new_index = pd.date_range(gdf.index[0], end=max_end, freq="1D")
```

Comment:
As you have observed, can you please updated `freq="1D"` to `freq="1H"`?

Reply:
Done. Thank you.